pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -2136,13 +2136,16 @@ public void monitorBacklogQuota() {
         });
     }
 
-    public boolean isTopicNsOwnedByBroker(TopicName topicName) {
-        try {
-            return pulsar.getNamespaceService().isServiceUnitOwned(topicName);
-        } catch (Exception e) {
-            log.warn("Failed to check the ownership of the topic: {}, {}", topicName, e.getMessage());
-        }
-        return false;
+    public CompletableFuture<Boolean> isTopicNsOwnedByBrokerAsync(TopicName topicName) {
+        return pulsar.getNamespaceService().isServiceUnitOwnedAsync(topicName)
```

Comment:
i notice that the sync version will at least with a timeout of `getMetadataStoreOperationTimeoutSeconds`. Maybe we need a timeout here.

Reply:
It seems that `isServiceUnitOwnedAsync` will always be bounded in the current implementations, so there doesn't seem to be a need to add another timeout solution.